Automation/Automation/Tests/Account.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Automation.Helpers;
+using Automation.Helpers.Enums;
+using Automation.Pages;
+
+namespace Automation.Tests
+{
+    [TestClass]
+    public class Account : BaseTest
+    {
+        [TestInitialize]
+        public override void Before()
+        {
+            base.Before();
+            Pages.HeaderPage.GoToAccountDropdownOption(AccountOption.LOG_IN);
+            Pages.LoginPage.Login(Constants.VALID_EMAIL, Constants.VALID_PASSWORD);
+        }
+        public static IEnumerable<object[]> BillingData()
+        {
+            yield return new object[]
+            {
+                new BillingInformation
+                {
+                    FirstName = Faker.Name.First(),
+                    MiddleName = Faker.Name.Middle(),
+                    LastName = Faker.Name.Last(),
+                    Email = Faker.Internet.Email(),
+                    Address = Faker.Address.StreetAddress(),
+                    City = Faker.Address.City(),
```

Comment:
The dynamic data here is not necessary, as it is only one flow. You can move the fakers in the BillinInformation object, and whenever you create a new instance of the class, the fields should be randomly generated.